client/src/context/userContext.jsx

```diff
@@ -0,0 +1,75 @@
+import {
+  createContext,
+  useEffect,
+  useState,
+  useCallback,
+  useContext,
+} from "react";
+import PropTypes from "prop-types";
+import Cookies from "universal-cookie";
+
+const UserContext = createContext(null);
+
+function UserProvider({ children }) {
+  const [user, setUser] = useState(null);
+  const cookies2 = new Cookies();
+  console.log(JSON.stringify(cookies2.get("userId")));
```

Comment:
Remove these two lines before merging, they look like they're just for testing

Reply:
oh yeah woops, that was testing code